ext/openssl/ossl_ssl.c

```diff
@@ -1502,6 +1502,31 @@ ossl_ssl_s_alloc(VALUE klass)
     return TypedData_Wrap_Struct(klass, &ossl_ssl_type, NULL);
 }
 
+static VALUE
+peer_ip_address(VALUE self) {
```

Comment:
Style: please follow the style of the surrounding code.

```suggestion
peer_ip_address(VALUE self)
{
```